src/Utility/Hash.php

```diff
@@ -446,14 +447,14 @@ public static function remove(array $data, $path)
      * following the path specified in `$groupPath`.
      *
      * @param array $data Array from where to extract keys and values
-     * @param string $keyPath A dot-separated string.
-     * @param string|null $valuePath A dot-separated string.
-     * @param string|null $groupPath A dot-separated string.
+     * @param string|array $keyPath A dot-separated string.
+     * @param string|array|null $valuePath A dot-separated string.
+     * @param string|array|null $groupPath A dot-separated string.
```

Comment:
I think `groupPath` should be `string|null` 
```php
static::extract($data, $groupPath)
```
extract just accept `string`

Reply:
I can try, but I think there are tests that pass an array of paths.